### PR TITLE
Cloud changelog - Dec 1st

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: source/install/cloud-changelog.md [hunk 1]
```diff
@@ -4,12 +4,53 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
-- [Release 2022-10-13](#release-2022-10-13)
+
+## Release 2022-12-01
+
+### Improvements
+
+#### User Interface (UI)
+ - Added message priority labels to the Threads view.
+ - Added support for enterprise users to request acknowledgements on posts and to acknowledge posts.
+ - Added a centralized page for draft messages.
```
Comment (amyblais) [inline source/install/cloud-changelog.md:21]: I can remove these if we're planning to keep the feature flags disabled for this cloud release.

Comment (esethna) [reply source/install/cloud-changelog.md:21]: Thanks @amyblais, we should plan to enable the feature flag for drafts and post priority for this cloud release (pending community testing and QA release approval)

**Review by mkraft — APPROVED** [review]

File: source/install/cloud-changelog.md [hunk 2]
```diff
@@ -4,12 +4,59 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
 - [Release 2022-11-10](#release-2022-11-10)
 - [Release 2022-10-27](#release-2022-10-27)
 - [Release 2022-10-20](#release-2022-10-20)
-- [Release 2022-10-13](#release-2022-10-13)
+
+## Release 2022-12-01
+
+### Highlights
+
+#### Message priority and Acknowledgments
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:18]:
```suggestion
#### Message Priority and Acknowledgments
```

File: source/install/cloud-changelog.md [hunk 3]
```diff
@@ -4,12 +4,59 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+ - Removed video check to allow the browser to decide what video types it can play.
+ - Added a tooltip to the right-hand side files filter icon.
+ - The number of users that can be added to a user group at once was increased to 256.
+ - Pre-packaged Boards v7.5.2.
+
+#### Administration
+ - Updated MacOS supported version to 11+.
+ - **My Insights** and OpenId Connect were added to the Free plan.
+ - All integration limits and usage limit components were removed.
+ - A notice banner in the **Invite** modal and at **system console > Site Statistics > Total Active Users** is now displayed to admins when the instance is in overage users state.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:37]:
```suggestion
 - A notice banner is displayed to admins on the **Invite** modal and on **System Console > Site Statistics > Total Active Users** page when the workspace exceeds the maximum number of users allowed.
```

Comment (cwarnermm) [reply source/install/cloud-changelog.md:37]: I'm not 100% sure my proposed update captures the correct state of the workspace for notification banners to appear based on "when the instance is in overage users state". This proposed update will benefit from a technical review.

File: source/install/cloud-changelog.md [hunk 4]
```diff
@@ -4,12 +4,59 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+ - For admins to be able to see how many license seats they have versus their total number of activated users, a **Paid Users** card was added to **system console > Team Statistics**.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:38]:
```suggestion
 - For admins to see how many license seats they have versus their total number of activated users, a **Paid Users** card was added to the **System Console > Team Statistics** page.
```

File: source/install/cloud-changelog.md [hunk 5]
```diff
@@ -4,12 +4,59 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+ - Added a new menu item in **system console > Users** to re-add users to all of their default teams and channels associated with the groups they're a member of.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:39]:
```suggestion
 - Added a new menu item on the **System Console > Users** page that re-adds users to all of their default teams and channels associated with the groups they're a member of.
```

File: source/install/cloud-changelog.md [hunk 6]
```diff
@@ -4,12 +4,59 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+
+### API Changes
+ - The resumable uploads API was exposed to plugins.
+ - Added a new API endpoint ``POST /api/v4/ldap/users/:user_id/group_sync_memberships`` to add (or re-add) users to all of their default teams and channels for all of the groups they're a member of.
+ - Added two new URL parameters to the ``GET /api/v4/groups`` endpoint to get the ``ChannelMemberCount`` for a group.
+ - Added new API endpoints ``POST /api/v4/users/:user_id/posts/:post_id/ack`` and ``DELETE /api/v4/users/:user_id/posts/:post_id/ack``.
+
+### Bug Fixes
+ - Fixed an issue where a blank message was displayed in threads if the leave/join messages were disabled.
+ - Fixed an issue where threads would appear duplicated in the threads list after leaving a channel.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:51]:
```suggestion
 - Fixed an issue where threads would appear duplicated in the Threads view after leaving a channel.
```

File: source/install/cloud-changelog.md [hunk 7]
```diff
@@ -4,12 +4,59 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+ - Fixed an issue with email search when using PostgreSQL database.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:52]:
```suggestion
 - Fixed an issue with email search when using a PostgreSQL database.
```

File: source/install/cloud-changelog.md [hunk 8]
```diff
@@ -4,12 +4,59 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+ - Fixed an issue where message drafts were not saved after a paste that went through the ``pasteHandler`` in the post textbox was not processed.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:53]:
```suggestion
 - Fixed an issue where message drafts were not saved after pasting them into the post textbox.
```

File: source/install/cloud-changelog.md [hunk 9]
```diff
@@ -4,12 +4,59 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+ - Fixed an issue where the team name in the channel sidebar header was not accessible.
+ - Fixed an issue where users were unable to open the profile popover from the right-hand side channel members list.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:55]:
```suggestion
 - Fixed an issue where users were unable to open the user's profile popover from the channel members list in the right pane.
```

File: source/install/cloud-changelog.md [hunk 10]
```diff
@@ -4,12 +4,59 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+
+### Known Issues
+ - Boards linked to a channel you're a member of doesn't automatically appear on your sidebar unless you're an explicit member of the board. As a workaround, you can access the board from the channel RHS, or by searching for the board via the board switcher (Ctrl/Cmd+K). Alternatively, you can ask the board admin to add you to the board as an explicit member. See the [issue-focalboard-4179](https://github.com/mattermost/focalboard/issues/4179) for more details.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:58]:
```suggestion
 - Boards linked to a channel you're a member of don't automatically appear on your sidebar unless you're an explicit member of the board. As a workaround, you can access the board from the channel RHS, or by searching for the board via the board switcher (Ctrl/Cmd+K). Alternatively, you can ask the board admin to add you to the board as an explicit member. See the [issue-focalboard-4179](https://github.com/mattermost/focalboard/issues/4179) for more details.
```

**Review by cwarnermm — COMMENTED** [review]
Provided feedback inline.

**Review by mickmister — APPROVED** [review]

File: source/install/cloud-changelog.md [hunk 11]
```diff
@@ -4,12 +4,61 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+ - Added a tooltip to the right-hand side files filter icon.
+ - The number of users that can be added to a user group at once was increased to 256.
+ - Keyboard and focus handling was improved in profile popovers and at-mentions.
+ - Pre-packaged Boards v7.5.2.
+
+#### Administration
+ - Updated MacOS supported version to 11+.
+ - **My Insights** and OpenId Connect were added to the Free plan.
+ - All integration limits and usage limit components were removed.
+ - A notice banner is displayed to admins on the **Invite** modal and on **System Console > Site Statistics > Total Active Users** page when the workspace exceeds the maximum number of users allowed.
```
Comment (johndavidlugtu) [inline source/install/cloud-changelog.md:38]: A global banner also pops up by the way. If number of actual users exceed number of paid users by less than 5%, the banner is dismissible. More than 10% and it's non-dismissible until the license seat count has been updated.

**Review by spirosoik — APPROVED** [review]

**Review by wuwinson — APPROVED** [review]

File: source/install/cloud-changelog.md [hunk 12]
```diff
@@ -4,12 +4,63 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+ - The number of users that can be added to a user group at once was increased to 256.
+ - Keyboard and focus handling was improved in profile popovers and at-mentions.
+ - Pre-packaged Boards v7.5.2.
+
+#### Administration
+ - Updated MacOS supported version to 11+.
+ - **My Insights** and OpenId Connect were added to the Free plan.
+ - All integration limits and usage limit components were removed.
+ - Team scheme APIs are now allowed to be administered with a Professional plan.
+ - A global banner as well as a notice banner are displayed to admins on the **Invite** modal and on **System Console > Site Statistics > Total Active Users** page when the workspace exceeds the maximum number of users allowed. If the number of actual users exceeds the number of paid users by less than 5%, the banner is dismissible. If the number of actual users exceeds the number of paid users by more than 10%, the banner is non-dismissible until the license seat count has been updated.
```
Comment (marianunez) [inline source/install/cloud-changelog.md:39]: This is only applicable for self-hosted, we can remove this for cloud

File: source/install/cloud-changelog.md [hunk 13]
```diff
@@ -4,12 +4,63 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+ - For admins to see how many license seats they have versus their total number of activated users, a **Paid Users** card was added to the **System Console > Team Statistics** page.
```
Comment (marianunez) [inline source/install/cloud-changelog.md:40]: Same here this is only for self-hosted v7.6

**Review by marianunez — APPROVED** [review]

**Review by dontoisme — APPROVED** [review]

**Review by johndavidlugtu — APPROVED** [review]

**Review by isacikgoz — APPROVED** [review]

**Review by gabrieljackson — APPROVED** [review]

File: source/install/cloud-changelog.md [hunk 14]
```diff
@@ -4,12 +4,51 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+ - Added new API endpoints ``POST /api/v4/users/:user_id/posts/:post_id/ack`` and ``DELETE /api/v4/users/:user_id/posts/:post_id/ack``.
+
+### Bug Fixes
+ - Fixed an issue where a blank message was displayed in threads if the leave/join messages were disabled.
+ - Fixed an issue where threads would appear duplicated in the Threads view after leaving a channel.
+ - Fixed an issue with email search when using a PostgreSQL database.
+ - Fixed an issue where message drafts were not saved after pasting them into the post textbox.
+ - Fixed an issue where the team name in the channel sidebar header was not accessible.
+ - Fixed an issue where users were unable to open the user's profile popover from the channel members list in the right panel.
+ - Fixed an issue where the OAuth 2.0 deprecation notice was still displayed in the system console.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:47]:
```suggestion
 - Fixed an issue where the OAuth 2.0 deprecation notice was still displayed in the System Console.
```

File: source/install/cloud-changelog.md [hunk 15]
```diff
@@ -4,12 +4,51 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+ - Added a new API endpoint ``POST /api/v4/ldap/users/:user_id/group_sync_memberships`` to add (or re-add) users to all of their default teams and channels for all of the groups they're a member of.
+ - Added two new URL parameters to the ``GET /api/v4/groups`` endpoint to get the ``ChannelMemberCount`` for a group.
+ - Added new API endpoints ``POST /api/v4/users/:user_id/posts/:post_id/ack`` and ``DELETE /api/v4/users/:user_id/posts/:post_id/ack``.
+
+### Bug Fixes
+ - Fixed an issue where a blank message was displayed in threads if the leave/join messages were disabled.
+ - Fixed an issue where threads would appear duplicated in the Threads view after leaving a channel.
+ - Fixed an issue with email search when using a PostgreSQL database.
+ - Fixed an issue where message drafts were not saved after pasting them into the post textbox.
+ - Fixed an issue where the team name in the channel sidebar header was not accessible.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:45]:
```suggestion
 - Fixed an issue where the team name in the channel sidebar header wasn't accessible.
```

**Review by cwarnermm — APPROVED** [review]
Feedback provided inline. Approving.

File: source/install/cloud-changelog.md [hunk 16]
```diff
@@ -4,12 +4,53 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+#### User Interface (UI)
+ - Removed video check to allow the browser to decide what video types it can play.
+ - Added a tooltip to the right-hand side files filter icon.
+ - The number of users that can be added to a user group at once was increased to 256.
+ - Keyboard and focus handling was improved in profile popovers and at-mentions.
+ - If a channel has a draft message, the pencil icon is now displayed on the right side of the channel name in the channel sidebar.
+ - Pre-packaged Boards v7.5.2.
+
+#### Administration
+ - Updated MacOS supported version to 11+.
```
Comment (justinegeffen) [inline source/install/cloud-changelog.md:27]:
```suggestion
 - Updated macOS supported version to 11+.
```

File: source/install/cloud-changelog.md [hunk 17]
```diff
@@ -4,12 +4,53 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+
+## Release 2022-12-01
+
+### Improvements
+
+#### User Interface (UI)
+ - Removed video check to allow the browser to decide what video types it can play.
+ - Added a tooltip to the right-hand side files filter icon.
+ - The number of users that can be added to a user group at once was increased to 256.
+ - Keyboard and focus handling was improved in profile popovers and at-mentions.
```
Comment (justinegeffen) [inline source/install/cloud-changelog.md:22]:
```suggestion
 - Keyboard and focus handling was improved in profile popovers and @mentions.
```

File: source/install/cloud-changelog.md [hunk 18]
```diff
@@ -4,12 +4,53 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+ - If a channel has a draft message, the pencil icon is now displayed on the right side of the channel name in the channel sidebar.
+ - Pre-packaged Boards v7.5.2.
+
+#### Administration
+ - Updated MacOS supported version to 11+.
+ - **My Insights** and OpenId Connect were added to the Free plan.
```
Comment (justinegeffen) [inline source/install/cloud-changelog.md:28]:
```suggestion
 - **My Insights** and OpenId Connect were added to the Mattermost Free plan.
```

File: source/install/software-hardware-requirements.rst [hunk 19]
```diff
@@ -32,7 +32,7 @@ Desktop apps
     :header: "Operating System", "Self-Hosted Technical Requirement", "Cloud Technical Requirement"
 
     "Windows", "Windows 8.1+", "Windows 8.1+"
-    "Mac", "MacOS 10.14+", "MacOS 10.14+"
+    "Mac", "MacOS 10.14+", "MacOS 11+"
```
Comment (justinegeffen) [inline source/install/software-hardware-requirements.rst:35]:
```suggestion
    "Mac", "macOS 10.14+", "macOS 11+"
```

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**